config/config.yaml

```diff
@@ -236,6 +248,9 @@ tide:
   - repos:
     - jetstack/tarmak
     - jetstack/cert-manager-csi
+    - jetstack/preflight
```

Comment:
Should we double check this repo (testing) is still included?

Reply:
Yes, we should! good spot.

Reply:
This has now been added